Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -137,10 +134,22 @@ class ClearPrivateDataTableViewController: UITableViewController {
         return tableView.dequeueReusableHeaderFooterViewWithIdentifier(SectionHeaderIdentifier) as! SettingsTableSectionHeaderView
     }
 
+    override func tableView(tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard section == SectionButton else { return nil }
+        let footer = tableView.dequeueReusableHeaderFooterViewWithIdentifier(SectionHeaderIdentifier) as! SettingsTableSectionHeaderView
```

Comment:
Nit: Now that we're using the header view as a footer it might make sense to rename it to SettingsTableSectionHeaderFooterView just to make it more clear.